debian/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM debian:stable-slim
+
+RUN apt-get update && \
+    apt-get install \
```

Comment:
Should we not install the AWS CLI for consistency? What about git - is it included in debian by default?

Reply:
Thanks, appears I indeed missed the AWS CLI and Git, doh. Will get these added.